src/commands/synthetics/utils.ts

```diff
@@ -793,8 +808,8 @@ export const renderResults = ({
 
     reporter.resultEnd(result, getAppBaseURL(config))
   }
-
-  reporter.runEnd(summary, getAppBaseURL(config))
+  const settings = await getOrgSettings(getApiHelper(config), reporter)
```

Comment:
I think we should move that to `execute` in `run-test.ts` so that `renderResults` only handles rendering